bin/compile

```diff
@@ -15,10 +15,11 @@ PROFILE_PATH="$BUILD_DIR/.profile.d/gsl.sh"
 
 mkdir -p $(dirname $PROFILE_PATH)
 mkdir -p $VENDOR_DIR
+mkdir -p "${HOME}/vendor"
```

Comment:
let's give this a name, like APP_VENDOR_DIR and change the existing VENDOR_DIR to BUILD_VENDOR or similar?

Reply:
GSL_BUILD_DIR = `build/vendor/gsl`
APP_VENDOR_DIR = `app/vendor`

Reply:
below, `tar xzf - --strip 1 -C GSL_BUILD_DIR`